server/sockets/sendMessage.js

```diff
@@ -1,56 +1,61 @@
-const {
-  NEW_EVENT_SEND_MESSAGE,
-  NEW_EVENT_SEND_FAILED,
-  NEW_EVENT_RECEIVE_MESSAGE,
-} = require('../../constants.json');
-const { addMessage, getActiveUser } = require('../utils/lib');
+const { NEW_EVENT_SEND_MESSAGE, NEW_EVENT_SEND_FAILED, NEW_EVENT_RECEIVE_MESSAGE } = require("../../constants.json");
```

Comment:
use the proper formatting, do a git pull, install prettier and update the result

Reply:
okie , I will update the code format with prettier !